aviary/subsystems/propulsion/test/test_propeller_performance.py

```diff
@@ -190,7 +190,8 @@ def setUp(self):
 
         pp = prob.model.add_subsystem(
             'pp',
-            PropellerPerformance(num_nodes=num_nodes, aviary_options=options),
+            PropellerPerformance(num_nodes=num_nodes,
+                                 aviary_options=options),
```

Comment:
remove?

Reply:
The interpolator builder still uses aviary_options. I am pretty sure we can remove it from there too, though that can be a nice compact follow-on.